saltlint/rules/NoTwoEmptyLinesRule.py

```diff
@@ -0,0 +1,22 @@
+# -*- coding: utf-8 -*-
+# Copyright (c) 2019 Dawid Malinowski <dawidmalina@gmail.com>
+
+from saltlint.linter import SaltLintRule
+
+class NoTwoEmptyLinesRule(SaltLintRule):
+    id = '217'
+    shortdesc = 'Files should not contain two consecutive blank lines.'
+    description = 'Files should not contain two consecutive blank lines.'
+    severity = 'LOW'
+    tags = ['formatting']
+    version_added = 'v0.1.1'
```

Comment:
Until code is merged into a version, we specify 'develop' for new features. A version number is added once a new release is made.

```suggestion
    version_added = 'develop'
```